s3/replication/replicator/tests/system/test_replicator.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# s3 replicator test file
+#
+# This file contains replicator tests
+# with various REST requests.
+
+import aiohttp
+import asyncio
+import os
+import sys
+import yaml
+from s3replicationcommon.log import setup_logger
+import pytest
+import json
+
+logger = None
+
+# Load the test job record.
+test_job = {}
+
+# String to save job_id of posted job
+job_uuid = ''
+
+
+@pytest.fixture()
+def jobfile(pytestconfig):
+    return pytestconfig.getoption("jobfile")
+
+
+@pytest.fixture()
+def configfile(pytestconfig):
+    return pytestconfig.getoption("configfile")
+
+
+@pytest.fixture
+def event_loop():
+    loop = asyncio.get_event_loop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture
+def setup_config(configfile, jobfile):
```

Comment:
may be name this test_config and we wont need params as you can infer file path as on line 70 below and then you wont need conftest.py

Reply:
done